frontend/src/app/features/test/login-test/login-test.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component } from '@angular/core';
 import { AuthService } from '../../../services';
-import { User } from 'firebase/auth';
+import { User } from '../../../models';
```

Comment:
We no longer to  use own `User` model.

We use user from `firebase/auth` instead. because It same interface.